src/usb_moded.c

```diff
@@ -686,7 +686,13 @@ void usbmoded_probe_init_done(void)
 {
     LOG_REGISTER_CONTEXT;
 
+#ifdef RESCUE
     usbmoded_set_init_done(access(usbmoded_init_done_flagfile, F_OK) == 0);
+    return;
+#endif
+
+    /* If rescue mode is not enabled, bypass init done check */
+    usbmoded_set_init_done(true);
```

Comment:
Would it be cleaner to have this in #else of that ifdef and remove the return above?